WebAPI/Controllers/UserController.cs

```diff
@@ -0,0 +1,55 @@
+using System.Collections.Generic;
+using FRITeam.Swapify.Backend.Interfaces;
+using FRITeam.Swapify.Entities;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace WebAPI.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/User")]
+    public class UserController : Controller
+    {
+        private ILogger<UserController> _logger;
+        private readonly IUserService _userService;
+
+        public UserController(ILogger<UserController> logger, IUserService userService)
+        {
+            _logger = logger;
+            _userService = userService;
+        }
+
+        // GET: api/User
+        [HttpGet]
+        public IEnumerable<string> Get()
+        {
+            return new string[] { "value1", "value2" };
+        }
+
+        // GET: api/User/5
+        [HttpGet("{id}", Name = "Get")]
+        public string Get(int id)
+        {
+            return "value";
+        }
+        
+        // POST: api/User
+        [HttpPost]
+        public void Post([FromBody]string value)
+        {
+        }
+        
+        // PUT: api/User/5
+        [HttpPut("{id}")]
+        public void Put(int id, [FromBody]User userToAdd)
+        {
+            _userService.AddAsync(userToAdd).Wait();
```

Comment:
This is not good example, you should check how to do it correctly

Reply:
ok